src/tests/ftest/scrubber/aggregation.yaml

```diff
@@ -39,7 +39,7 @@ pool:
 container:
   type: POSIX
   control_method: daos
-  oclass: RP_2G1
+  oclass: RP_2GX
```

Comment:
Why set the container default oclass to RP_2GX but then IOR oclass to RP_2G1?

Reply:
I noticed dfs_dir_oclass is set to either RP_2G1 or RP_2GX on container level is used in ior command. ior command fails with Input/Output error when we set RP_2G1 to dfs_dir_oclass on ior command for this testing.

Presently, ior uses dfs.dir_oclass as RP_2GX and dfs_oclass as RP_2G1. Now, ior command works fine.

Reply:
> ior command fails with Input/Output error when we set RP_2G1 to dfs_dir_oclass on ior command for this testing

That doesn't make sense. IOR completely ignores `dfs_dir_oclass`.

Reply:
I don't know... Maybe, that an area which needs to be investigated later while address DAOS-14661.

Reply:
I suspect there are probably bugs with scrubber or aggregation related to G1 vs GX.
When you set `oclass: RP_2GX` at the _**container level**_, the **_default_** directory and file object classes, as well as the DFS root object, will use RP_2GX.
Then, when you tell IOR to use `--dfs.oclass RP_2G1`, _**only files**_ will be using RP_2G1. Which means in total, the DFS root object will use RP_2GX and files will use RP_2G1.